### ISSUE TITLE
compability with Elixir 1.5

### ISSUE THREAD

**Issue by SemmelK** [antonmi/espec#211]
Hello,
it seems to me that espec is not compatible with Elixir's current version (1.5). I get the following error on `mix espec:` 

`(UndefinedFunctionError) function ExUnit.CaptureServer.start_link/0 is undefined or private. Did you mean one of:
      * start_link/1
    ExUnit.CaptureServer.start_link()
    (espec) lib/mix/tasks/espec.ex:139: Mix.Tasks.Espec.run_espec/3
    (espec) lib/mix/tasks/espec.ex:121: Mix.Tasks.Espec.run/1
    (mix) lib/mix/task.ex:301: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:75: Mix.CLI.run_task/2`

ExUnits has changed the interface to CaptureServer.start_link, so you probably have to add empty options to its call.

**Comment by antonmi** [comment]
Thanks, @SemmelK !
Will take a look!
If you have a solution, please do not hesitate with PR!)

**Comment by SemmelK** [comment]
I could try a PR but I'm not sure if the new version will be compatible with Elixir Version < 1.5. I need to replace this 
`ExUnit.CaptureServer.start_link()`
with
`ExUnit.CaptureServer.start_link([])`
in lib/espec.ex

I can also check the arity of start_link like this:
```
case ExUnit.CaptureServer.__info__(:functions)[:start_link] do
   1 -> ExUnit.CaptureServer.start_link([])
    _ -> ExUnit.CaptureServer.start_link()
 end
```

but this will result in compile warnings for all versions of Elixir.

**Comment by antonmi** [comment]
May we check Elixir version in compile time?

**Comment by SemmelK** [comment]
I have created a PR with a compile time version check.

**Comment by antonmi** [comment]
@SemmelK 
I've just published 1.4.2!
Thanks for using ESpec

**Comment by SemmelK** [comment]
Great, thanks for your fast responses and this awesome testing framework!
